tests/test_limoss.py

```
@@ -0,0 +1,262 @@
+
+        # [cmd=0x02, button_count=8, system_type=0x12 (2 motors), vibration=1, mem=5]
+        packet = controller._build_packet(LimossCommands.QUERY_CAPABILITIES, 8, 0x12, 1, 0x05)
+        controller._handle_notification(MagicMock(), bytearray(packet))
+
+        assert controller.memory_slot_count == 5
+
+    async def test_read_positions_queries_each_motor(
+        self,
+        hass: HomeAssistant,
+        mock_limoss_config_entry,
+        mock_coordinator_connected,
```

Ruff (0.14.14)

[warning] Unused method argument: `mock_coordinator_connected`

(ARG002)